docs/usage.md

```diff
@@ -93,7 +93,8 @@ To turn on digital normalization, use the `--bbnorm` parameter and, if required,
 
 By default, the pipeline uses Megahit (`--assembler megahit`) to assemble the cleaned and trimmed reads to create the reference contigs.
 Megahit is fast and it does not require a lot of memory to run, making it ideal for large sets of samples.
-The workflow also supports RNAspades, (`--assembler rnaspades` ) as an alternative.
+The workflow also supports Spades, (`--assembler spades` ) as an alternative.
+If you work with virus you can specify it into SPADES by using the option `--spades_flavor rnaviral`
```

Comment:
```suggestion
The workflow also supports Spades (`--assembler spades` ) as an alternative.
If you work with virus RNA you can specify that by using the Spades option `--spades_flavor rnaviral` (see [parameter documentation](/metatdenovo/parameters/#spades_flavor))
```

This links directly to the respective parameter on the website. (See [example link](https://github.com/nf-core/funcscan/blob/master/docs/usage.md?plain=1#L28) from funcscan)